cdap-api/src/main/java/co/cask/cdap/api/metadata/MetadataEntity.java

```diff
@@ -75,9 +75,10 @@
   public static final String ARTIFACT = "artifact";
   public static final String VERSION = "version";
   public static final String DATASET = "dataset";
-  public static final String DATASET_INSTANCE = "DatasetInstance";
+  public static final String V1_DATASET_INSTANCE = "datasetinstance";
```

Comment:
Why do we need to add it here? This exposed it to programmatic API as an option to use we should not allow it.